doc/nrf/includes/sample_dtm_radio_test_fem.txt

```diff
@@ -0,0 +1,36 @@
+
+You can add support for the nRF21540 front-end module to the sample.
+Set ``-DNRF5340EK_FEM=y`` when you invoke ``west build`` or ``cmake`` in your sample directory.
+Follow the instructions in :ref:`cmake_options`.
+For example:
+
+``west build -b nrf5340dk_nrf5340_cpunet -- -DNRF21540EK_FEM=y``
+
+When you set ``NRF5340EK_FEM=y`` the :file:`dts-nrf21540-fem.overlay` devicetree overlay file is used to override the default settings.
+The configuration from this file is merged with the additional development kit overlay file from the :file:`boards` folder if it exists.
+
+The :file:`dts-nrf21540-fem.overlay` is located in the |fem_file_path| folder.
+Make sure that the GPIOs and the SPI configuration in the file correspond to those in which your front-end module is connected.
+
+.. note::
+   You must set ``NRF5340EK_FEM=y`` if you use the nRF21540 EK.
+   If you use the nRF21540 DK, build your application for the :ref:`nrf21540dk_nrf52840 <zephyr:nrf21540dk_nrf52840>` board.
+   The devicetree for the nRF21540 DK already contains the required FEM configuration, so you do not need to set additional build option.
+
+Sometimes you may need to add additional overlay file.
```

Comment:
```suggestion
Sometimes, you may need to add an overlay file.
```
Should we elaborate what "sometimes" mean. In which situations?
If the needed overlay file is something on top of an existing one, you could say "extra" instead of additional. (Because add additional does not really sound good)

Reply:
Done, I added the "extra" instead of additional. Sometimes" I mean here that it depends on the user needs but maybe this word is no needed here. What do you think ?

Reply:
You might want to give an example case when the user would need to add the overlay file.

Reply:
What about: "Your application may need to add an extra overlay file". Example case is hard to invent because it only depends on the user needs some user can have need to add such file